services/spar/test-integration/Util/Scim.hs

```diff
@@ -108,7 +108,7 @@ randomScimUserWithSubject ::
   (HasCallStack, MonadRandom m) =>
   m (Scim.User.User SparTag, SAML.UnqualifiedNameID)
 randomScimUserWithSubject = do
-  fieldCount <- getRandomR (0, 3)
+  fieldCount <- getRandomR (1, 3)
```

Comment:
we should quickcheck effectful code too ... nothing stopping us from doing that honestly